app/assets/stylesheets/components/cards.scss

```diff
@@ -5,6 +5,7 @@
   background: var(--card-bg);
   color: var(--card-color);
   box-shadow: 0 0 0 1px var(--card-border);
+  word-break: break-word;
```

Comment:
Hey @Link2Twenty !

Just dropping [this (mdn word-break docs)](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) here.

Quotes about `word-break: break-word`:
* "This deprecated API should no longer be used, but will probably still work"
* Solution: "Has the same effect as `word-break: normal` and `overflow-wrap: anywhere`, regardless of the actual value of the `overflow-wrap` property."

Reply:
Good catch 😅